src/main/java/tatracker/model/module/Module.java

```diff
@@ -123,6 +123,34 @@ public boolean hasDoneSession(Session session) {
         return doneSessions.contains(session);
     }
 
+    /**
+     * Sorts students in the groups alphabetically.
+     */
+    public void sortGroupsAlphabetically() {
+        for (int i = 0; i < groups.size(); ++i) {
+            groups.get(i).sortStudentsAlphabetically();
+        }
+    }
+
+    /**
+     * Sorts the students in the groups by rating in ascending order.
+     */
+    public void sortGroupsByRatingAscending() {
+        for (int i = 0; i < groups.size(); ++i) {
+            groups.get(i).sortStudentsByRatingAscending();
+        }
+    }
+
+    /**
+     * Sorts the students in the groups by rating in descending order.
+     */
+    public void sortGroupsByRatingDescending() {
+        for (int i = 0; i < groups.size(); ++i) {
+            groups.get(i).sortStudentsByRatingDescending();
+        }
+    }
```

Comment:
Cool! In future though enhanced for loop is preferred!

Reply:
The for (Group group : groups) !

Reply:
Ohh, ok can! I'll make all the changes you mentioned here when I make a PR to create the default view function?